android/src/main/java/com/thebluealliance/androidclient/binders/TeamInfoBinder.java

```diff
@@ -118,6 +125,13 @@ public void updateData(@Nullable TeamInfoBinder.Model data) {
             string.setSpan(new TextAppearanceSpan(mActivity,
                     R.style.InfoItemLabelStyle), 0, 3, Spannable.SPAN_INCLUSIVE_INCLUSIVE);
             teamFullName.setText(string);
+            teamFullName.setMaxLines(TEAM_FULL_NAME_COLLAPSED_MAX_LINES);
+            teamFullName.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
I'm not totally comfortable with hiding the ellipsis in the middle of the text. I think people will expect to see it at the end of the content.